src/main/java/me/ironexception/universalfriends/configuration/Operations.java

```diff
@@ -12,19 +14,52 @@
 
 public class Operations {
 
-    public static <T extends IPerson> Set<T> getByFriendliness(Configuration<T> configuration, double value) {
+    /**
+     * Gives you a set of all players with that specific value.
+     * WARNING: This should not be used to get all friends or something like that. It's only there for completeness
```

Comment:
Don't refer to the reader in javadocs. The same goes as mentioned before: javadocs must be precise and correct, without unecessary information.

That means the entire warning has to go, and change the first line to something along the lines of:
```
Produces a immutable {@link Set} of all persons whose friendliness value is equal to the provided value.
```

Reply:
Why not the warning tho?

Reply:
Because the javadoc describes what a method does and not when you should use it or why it exists.